### DISCUSSION TITLE
Need help with using custom font

### DISCUSSION THREAD

**Discussion by chickenfeetsd** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3310] [General]
I'm fairly new to the library. Is that possible to setup the MaterialDesignThemes (v4.8.0) library to use different font other than Roboto. If so, can you please point me to the tutorial if it exists. Thank you very much.

**Comment by Keboo** [comment]
Yea it is certainly possible, though this is more of a WPF thing. The library comes with Roboto bundled in, but you can swap it out for something else. 

A good first step is to simply set whatever font you want at your Window. By just setting the [FontFamily property](https://learn.microsoft.com/dotnet/api/system.windows.controls.textblock.fontfamily) to whatever font you like.

This is largely what the library is doing for you when you set the Window style (as seen [here](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/master/MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Window.xaml#L6)).

**Comment by chickenfeetsd** [comment]
Thanks Kevin for the response. I'm not sure if this is going to work. I've tried several approaches to set the font family to use Arial, however, material design library doesn't take in the new font and applies to its controls.
Approach #1: Set font family to use Arial in MainWindow constructor
Approach #2: Override MaterialDesignWindow style but setting font family to use Arial
Did I perhaps missing anything here?

**Comment by chickenfeetsd** [comment]
I got the custom font to work finally. Thanks Kevin.
